db/fixtures/miq_policy_sets.yml

```diff
@@ -105,7 +105,24 @@
           description: Initiate SmartState Analysis for Container Image
           action_type: default
           options: {}
-      Condition: []
+      Condition:
+        - name: do not scan image-inspectors image
```

Comment:
@cban, the first one. I couldnt add the ' character in that file...
On Jul 4, 2016 11:59 AM, "Beni Cherniavsky-Paskin" <notifications@github.com>
wrote:

> In db/fixtures/miq_policy_sets.yml
> <https://github.com/ManageIQ/manageiq/pull/9484#discussion_r69427860>:
>
> > @@ -105,7 +105,24 @@
> >            description: Initiate SmartState Analysis for Container Image
> >            action_type: default
> >            options: {}
> > -      Condition: []
> > +      Condition:
> > +        - name: do not scan image-inspectors image
>
> image-inspector's image or just image-inspector image
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/9484/files/32e2f45b48230318c74866bb8368ce8bf5b89a50#r69427860>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AC-ogEoWdCISp5FjS99xnYtrsHv4w1tvks5qSMtlgaJpZM4JAEB7>
> .
>